src/styles/class-overrides.scss

```diff
@@ -0,0 +1,128 @@
+.usa-prose,
+.usa-intro,
+.font-heading-xs,
+.font-heading-xl,
+.text-primary,
+.usa-graphic-list__heading,
+.usa-hero__heading {
+  font-family: family("ui");
+}
+
+.usa-nav__submenu,
+.usa-nav__primary button[aria-expanded="true"],
+.usa-hero__callout,
+.usa-section--dark {
+  background-color: color("base-darker");
+}
+
+.usa-nav__primary button[aria-expanded="false"] {
+  color: color("disabled-dark");
+  &:hover {
+    color: color("base-darker");
+  }
+  &::after {
+    border-style: solid;
+    border-width: $mocto-button-border-width;
+    border-color: color("secondary");
+  }
+}
+
+.usa-nav__primary > .usa-nav__primary-item > a {
+  color: color("disabled-dark");
+  &:hover {
+    color: color("base-darker");
+  }
+}
+
+.usa-button {
+  color: color("base-darker");
+  &:hover {
+    color: color("base-darker");
+    background-color: color("primary");
+    border-style: solid;
+    border-width: $mocto-button-border-width;
+    border-color: color("base-darker");
+  }
+  &:active {
+    color: white;
+    background-color: color("base-darker");
+  }
+  &:disabled {
+    color: white;
+    background-color: color("disabled");
+  }
+}
+
+.usa-button--accent-warm {
+  color: color("base-darker");
+  background-color: color("secondary");
+  &:hover {
+    color: color("base-darker");
+    background-color: color("secondary");
+    border-style: solid;
```

Comment:
I noticed a convention in how you order it `color background-color border-style`, where is this from?

Reply:
I'll look into conventions. I just did text/color related first then border related but didn't consult any documentation regarding ordering